src/DynamoCoreWpf/ViewModels/VisualizationSettingsViewModel.cs

```diff
@@ -1,39 +1,37 @@
-﻿using System;
+﻿using Dynamo.Interfaces;
 using Dynamo.ViewModels;
 
 namespace Dynamo.Wpf.ViewModels
 {
-    public class VisualizationSettingsViewModel
+    public class RenderPackageFactoryViewModel : ViewModelBase
```

Comment:
This view model is now renamed and wraps the `RenderPackageFactory`. This is done to avoid holding a reference to the `DynamoViewModel` which was causing a circular reference and a binding leak.